source/partials/_nav.slim

```diff
@@ -5,8 +5,6 @@ nav.main-nav
     li
       a href="/gems" Gems
     li
-      a href="http://github.com/dry-rb" GitHub
+      a href="/community" Community
     li
-      a href="https://discuss.dry-rb.org" Discuss
```

Comment:
Yes, I figured it would be better to have a simpler top-level nav, and we link to all the community interactions points (including the forum) over on the community page anyway.

I think there's a couple of things we could do to raise the prominence of links to the forum:

- Make the links in the top paragraph on the community page bigger, bolder and more obvious. Start by just bolding them?
- Build out an extra nav in the footer that can link to all these specific items (perhaps have a simple, vertical list of links replace the chunky github/twitter/gitter "buttons")